src/com/mambu/apisdk/services/LoansService.java

```diff
@@ -131,6 +132,12 @@
 		// Use LoanAccountPatchJsonSerializer to make the expected format
 		patchAccount.addJsonSerializer(LoanAccount.class, new LoanAccountPatchJsonSerializer());
 	}
+	// Patch Account. Used to link loan accounts with savings accounts
+	private final static ApiDefinition postSettlementForLoanAccount;
+	static {

```

Comment:
Initially we needed it when we had to set also the serialiser but not anymore. I`ll remove it. Thanks!!